identity-service/sequelize/migrations/20220512172226-add_notification_json_b.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    // This migration requires each enum add in it's own query, otherwise Sequelize complains
+    // that this can't be performed within a transaction
+    return queryInterface.sequelize.query(
+      `ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'TipSend'`)
+      .then(() =>
+        queryInterface.sequelize.query(`ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'TipReceive'`)
+      ).then(() =>
+        queryInterface.sequelize.query(`ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'Reaction'`)
+      ).then(() =>
+        queryInterface.sequelize.query(`ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'SupporterRankUp'`)
+      ).then(() =>
+        queryInterface.sequelize.query(`ALTER TYPE "enum_SolanaNotifications_type" ADD VALUE 'SupportingRankUp'`)
+      ).then(() => queryInterface.addColumn('SolanaNotifications', 'metadata', {
+        type: Sequelize.JSONB,
+        allowNull: true
+      })).then(() =>
+        queryInterface.sequelize.query(
+          `CREATE INDEX IF NOT EXISTS "solana_notifications_metadata_tip_tx_signature_idx" ON "SolanaNotifications"(("metadata"->>'tipTxSignature'))`
+        )
+      )
+  },
+
+  down: (queryInterface, Sequelize) => {
```

Comment:
nice adding this down, interesting pattern too

Reply:
such a pain 😭